clang/runtime/dpct-rt/include/dpct/bindless_images.hpp

```diff
@@ -228,12 +229,13 @@ dpct_memcpy(void *src, sycl::ext::oneapi::experimental::image_mem_handle dest,
   const auto dest_offset =
       sycl::range<3>(w_offset_dest / ele_size, h_offset_dest, 0);
   const auto copy_extend = sycl::range<3>(w / ele_size, h, 0);
-  return q.ext_oneapi_copy(src, src_offset, src_extend, dest, dest_offset,
-                           desc_dest, copy_extend);
+  return q.ext_oneapi_copy(const_cast<void *>(src), src_offset, src_extend,
```

Comment:
It's better to ask CodePlay to refine the API interface.

Reply:
Add TODO.